packages/suite/src/utils/suite/stake.ts

```diff
@@ -618,3 +618,38 @@ export const getChangedInternalTx = (
 
     return internalTransfer ?? null;
 };
+
+export const simulateUnstake = async ({
+    amount,
+    from,
+    symbol,
+}: StakeTxBaseArgs & { amount: string }) => {
+    const ethNetwork = getEthNetworkForWalletSdk(symbol);
+    const { address_pool: poolAddress, contract_pool: contractPool } = selectNetwork(ethNetwork);
+
+    if (!amount || !from || !symbol) return null;
+
+    const amountWei = toWei(amount, 'ether');
+    const interchanges = 0;
+    const coin = symbol?.toString();
+
+    const data = contractPool.methods
+        .unstake(amountWei, interchanges, WALLET_SDK_SOURCE)
+        .encodeABI();
+    if (!data) return null;
+
+    const ethereumData = await TrezorConnect.ethereumCall({
+        coin,
```

Comment:
no reason to do `symbol?.toString()`
```suggestion

    const data = contractPool.methods
        .unstake(amountWei, interchanges, WALLET_SDK_SOURCE)
        .encodeABI();
    if (!data) return null;

    const ethereumData = await TrezorConnect.ethereumCall({
        coin: symbol,
```

Reply:
done, https://github.com/trezor/trezor-suite/pull/12122/commits/9e1a100bab253b89e9c8af0fbe2e3fb7e20f24aa